.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What are we updating (eg. update button destructive color)?'
+        default: ''
+        required: true
+      jira:
+        description: 'What is the related JIRA (eg. SW-1234)?'
+        default: ''
+        required: true
+      email:
+        description: 'Your terraformation email'
+        default: ''
+        required: true
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Create branch
+        run: git checkout -b `echo design-workflow-${{ github.actor }}-${{ github.event.inputs.jira }}-\`date +"%y-%m-%d"\`  | sed "s/ /-/g"`
+
+      - name: Install Dependencies
+        run: yarn install --network-timeout 100000
+
+      - name: Build dictionary 
+        run: yarn build-dictionary
+
+      - name: Bump package patch version
+        run: yarn bump patch
+
+      - name: Config user
+        run: |
+          git config --global user.email ${{ github.event.inputs.email }}
+          git config --global user.name ${{ github.actor }}
+
+      - name: Commit changes
+        run: |
+          git add -A
+          git commit -m "${{ github.event.inputs.description }}"
```

Comment:
Probably worth testing what this does if the user puts double quotes or dollar signs in the description.

Reply:
Good catch, it does break. 
```
/home/runner/work/_temp/ca28[9](https://github.com/karthikbtf/test/runs/8118436180?check_suite_focus=true#step:9:10)fac-e0b4-478b-b3e9-0ea401b933f2.sh: line 2: unexpected EOF while looking for matching `"'
```
I need to sanitize user input.. hmm,, there's a long list of special shell characters like ! % etc.. let me look online for a comprehensive sanitization code.

Reply:
If you take that approach, you could write the PR title/description as the first line of the commit message, and then tell `gh pr create` to pull the PR info from the commit message. Otherwise you'll have the same problem with special characters in the `gh` command line, I think.

Reply:
> Alternate idea: write the description to a file and use `git commit -F` to read the commit message from there. Should be less funny business with special characters that way.

Let me try that, I have to use some command other than `echo` to write to file.

Reply:
Actually echo won't be an issue here.

Reply:
File suggestion worked great, using a simple node script to write to file since echo tries to interpret $. Will update this PR.

<img width="960" alt="Screen Shot 2022-08-31 at 10 44 54 AM" src="https://user-images.githubusercontent.com/1865174/187745070-f637d5d2-7c6a-4b62-b61e-e6cbfd4ff868.png">